app/components/DocumentListItem.tsx

```diff
@@ -49,6 +52,17 @@ function DocumentListItem(
   const user = useCurrentUser();
   const [menuOpen, handleMenuOpen, handleMenuClose] = useBoolean();
 
+  let itemRef: React.Ref<HTMLAnchorElement> =
+    React.useRef<HTMLAnchorElement>(null);
+  if (ref) {
+    itemRef = ref;
+  }
+
+  const [tabIndex, focused, handleKeyDown, handleClick, handleFocus] =
```

Comment:
Why would this library return a massive array instead of an object? Very tempted to change that seeing as we've forked it already.

Reply:
Makes sense 👍